src/store/wallet.ts

```diff
@@ -371,6 +379,7 @@ export const actions = actionTree(
     async requestTransactionsHistory({ commit, getters }, { force = false, offset = 0 }): Promise<ZkInTx[]> {
```

Comment:
It's a recommendation:
think about the option to store offset inside and while calling check stored instead of saving this logic outside to transaction history request